### PR TITLE
 Write script to backfill Law enforcement number

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/11700

This code includes the backfilling script as well as a migrations file to update the datatype for law enforcement num from int to text.

The spreadsheet i got didn't have `crash_id`s so i used a combination of `case_id` and `crash_date` as an extra precaution to target the correct record. Though I dont think it was even necessary to use date bc there were 25 `case_id` duplicates in the script and they all resulted from fatalities that were in the same crash.

## Testing
**URL to test:** 
[Staging](https://visionzero-staging.austinmobility.io/editor/#/fatalities)

**Steps to test:**
1. I already tested this locally and on staging and looks like the script worked!
2. There's not much else for y'all to test besides checking out the staging website and seeing there are now law enforcement nums on the Fatalities page. Sort descending by crash date to see the bulk of them, for staging there were 419 crashes updated with law enforcement nums. For prod there will be more.
3. If you really want to go above and beyond you can run the script on your local DB and I can send you the spreadsheet.

---
#### Ship list
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-toolbox/ftt_backfill/law_enforcement_num_backfill.py
Comment (roseeichelmann) [inline atd-toolbox/ftt_backfill/law_enforcement_num_backfill.py:1]: this whole file is for the purpose of changing the datatype of `law_enforcement_num` in the `atd_txdot_crashes` table from int to text

File: atd-vzd/migrations/migration_crashes_view_fatalities_2023_04_05--1642.sql
Comment (roseeichelmann) [inline atd-vzd/migrations/migration_crashes_view_fatalities_2023_04_05--1642.sql:1]: this whole file is for the purpose of changing the datatype of law_enforcement_num in the atd_txdot_crashes table from int to text

**Review by patrickm02L — APPROVED** [review]
Good to go! 🚀

**Review by mddilley — APPROVED** [review]
Good to go! Read the code, and I checked out the law enforcement numbers in VZE staging too. Thanks for getting these in the DB! 🙏 🚢

**Pull request merged**